server/src/dbschema.js

```diff
@@ -1,8 +1,19 @@
 const db = require("./db");
 const Keygrip = require('keygrip');
+const pgpatcher = require("pg-patcher");
+
+// MAKE SURE TO SET THE PATCH LEVEL TO THE HIGHEST LEVEL
+// IN THE STATEMENT INSERT INTO property(key, value) VALUES('patch', 2);
 
 const createSQL = `
```

Comment:
Hmm, yes, I added the INSERT INTO at the end without thinking about it too much...